prevent-broker-scale-down-if-contains-partition .md

```diff
@@ -0,0 +1,63 @@
+
+# Prevent scale down of the brokers if they contain partitions
+
+This proposal is about adding the mechanism for preventing the scale down of the brokers in the cluster if a partition is assigned to it.
+In the future, when we introduce the automatic scale up/down of the Kafka Cluster, this mechanism would help to prevent any data loss when we are scaling down the brokers when they have some partitions assigned to them.
```

Comment:
Which "automatic scale up/down" are you referring to?

Reply:
Oh my bad, Let me address this

Reply:
I was actually trying to point towards the next steps of the Automatic add/ remove of the brokers from cluster. Like cureently we have introduced modes in the rebalance but in future we were going to do all the process without user interaction

Reply:
so more than "automatic scale up/down of the Kafka Cluster" is "automatic rebalancing when scaling up/down the Kafka cluster". I would also link to the [proposal](https://github.com/strimzi/proposals/blob/main/035-rebalance-types-scaling-brokers.md) where we mention this future plan, so that people know what is in place.